docs/source/user_guide/tutorials/evaluator.rst

```diff
@@ -63,3 +63,5 @@ For the math-solver example, we will use LLM-as-a-judge to be the evaluator. The
       return assess.score
 
 The evaluator agent uses `gpt-4o-mini` as the backbone model. It also returns a structured output, ``Assessment``, to enforce the output format since we require the evaluator to return a numerical value.
+
+Cognify provides a few `sample evaluators <https://github.com/GenseeAI/cognify/tree/main/cognify/optimizer/evaluation>`_ to start with: F1 score, LLM-as-a-judge, exact match, and code execution for the HumanEval dataset used in our examples.
```

Comment:
"for you to start with", including xxxx code execution. No need to include "for the humaneval" part, as it could confuse readers that all these sample evaluators are for the humaneval workload.

Also, I think this sentence could go after the first para, before the example.

Reply:
Done